### ISSUE TITLE
Several Carnivores chase Prey, but will not attack them.

### ISSUE THREAD

**Issue by TyrannosaurKing** [FossilsArcheologyRevival/FossilsArcheologyRevival#1073]

**Comment by TyrannosaurKing** [comment]
I've tested this with Rex and Dilo, Rex will chase Dilo until the end of time but not attack it. Veloci and Deino do the same with things such as an infant Plesiosaur and Itchyosaur, they just kinda move around it but won't do anything with it.

**Comment by TyrannosaurKing** [comment]
Okay after some time, Rex randomly decided to Attack Dilo, but only after some time had passed, tested it again and Rex is doing what Veloci/Deino does with Plesi and Itchy, going twords the Dilo, Walking around it, running off then coming back and doing the same. Either it Eventually Kills the Dilo, or just walks off all-together.

**Comment by lindstheginj** [comment]
If they are killing the animals, then is there really a problem?

**Comment by lindstheginj** [comment]
This should have been fixed in 8.0.1. @TyrannosaurKing are you still experiencing the issue?
